### DISCUSSION TITLE
syslog messages do not parse correctly

### DISCUSSION THREAD

**Discussion by iron20** [Security-Onion-Solutions/securityonion#8422] [Unsupported Versions] [answered]
Hello guys,
I have a standalone SO installation, I have a cisco switch and router that send syslog messages to the manager, but elasticsearch somehow fails to parse them correctly, is there any solution other then creating a custom parsing pipeline? if that's the case can someone guide me on how exactly i can achieve such thing (I'm a security analyst intern and ive been stuck in this problem for 3 weeks now, any clue or advice will highly be appreciated)

![image](https://user-images.githubusercontent.com/72130871/182121996-c18aaf6a-78e0-4d0e-8de6-d9b71f036142.png)

**Comment by iron20** [comment]
> Hello guys, I have a standalone SO installation, I have a cisco switch and router that send syslog messages to the manager, but elasticsearch somehow fails to parse them correctly, is there any solution other then creating a custom parsing pipeline? if that's the case can someone guide me on how exactly i can achieve such thing (I'm a security analyst intern and ive been stuck in this problem for 3 weeks now, any clue or advice will highly be appreciated)
> 
> ![image](https://user-images.githubusercontent.com/72130871/182121996-c18aaf6a-78e0-4d0e-8de6-d9b71f036142.png)

![image](https://user-images.githubusercontent.com/72130871/182123966-a7458fd2-6684-42f6-96c4-38c238d3179b.png)
 and even windows wazuh agent logs dont get parsed

**Comment by reyesj2** [comment, marked as answer]
As I am reading and rereading I think you're looking to have the fields log.source.address and message shown without clicking the 'expand document button' in kibana

This is because the default fields in the unexpanded view don't include 'log.source.address' or 'message'.

You can add whatever field you'd like to your unexpanded view by clicking the '+' next to whatever field you want to show up. In your case next to log.source.address and click 'toggle column in table'.
You can repeat that for every column you want to show up on your overview page without expanding individual documents.

**Reply by TotieBash** [reply]
Yes, I would also I add that any modification you do with the default SecurityOnion Dashboard is not persistent.  I would "clone" the particular dashboard you want and make your modification on the cloned dashboard to make it persistent.

Your use of the term "parse" is not accurate.  I thought you needed filebeat cisco.ios module.  This module parses syslog's one big blob "message" and enrich it to more fields.  If you want to tackle that then I would refer you to this Netflow how to:  https://www.youtube.com/watch?v=ew5gtVjAs7g&t=704s
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-cisco.html
![image](https://user-images.githubusercontent.com/86748904/182211588-6bea7875-7160-49af-a79d-40f3137d12f2.png)

**Reply by iron20** [reply]
So i was trying to parse logs that were already parsed but the fields in the table weren't accurate, thank you guys for the help, i'll try to clone the dashboard in order to make it persistent hopefully i wont have a problem with that as well.

**Reply by iron20** [reply]
> Yes, I would also I add that any modification you do with the default SecurityOnion Dashboard is not persistent. I would "clone" the particular dashboard you want and make your modification on the cloned dashboard to make it persistent.
> 
> Your use of the term "parse" is not accurate. I thought you needed filebeat cisco.ios module. This module parses syslog's one big blob "message" and enrich it to more fields. If you want to tackle that then I would refer you to this Netflow how to: https://www.youtube.com/watch?v=ew5gtVjAs7g&t=704s https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-cisco.html ![image](https://user-images.githubusercontent.com/86748904/182211588-6bea7875-7160-49af-a79d-40f3137d12f2.png)

i figured out how to "clone" dashboards and created couple ones for each event module so i get pretty looking interfaces,thank you for the tips
